### PR TITLE
Fix(DB/creature_loot_template) Skinning loot table from TrinityCore

### LINKED ISSUE #5821: Dustwallow Marsh: Firemane mobs not consistently dropping leather
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [X] enUS
- [ ] other (specify)

##### FACTION

- [X] Alliance
- [ ] Horde

##### CONTENT PHASE:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [X] 30-39
- [X] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

"Originally reported https://github.com/chromiecraft/chromiecraft/issues/632"

Firemane Scout, Scalebane, Flame Caller, and Ash Tail are not consistently dropping leather.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

There should always be a leather drop.

##### STEPS TO REPRODUCE THE PROBLEM:

1. Kill a Firemane mob
2. Skin

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/6947789622de45479d2311fd5d1635af3a957b5d

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

Confirmed and reproduced on a local AC server :

WoWScrnShot_051321_115739
WoWScrnShot_051321_115814
WoWScrnShot_051321_115826

Some of them have loot:

WoWScrnShot_051321_115750

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/98542816-dustwallow-marsh-firemane-mobs-not-consistently-dropping-leather?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Branel**
## Changes Proposed:
- Adjust drop rate for ashmane dragons 
## Issues Addressed:

- Closes #5821

## SOURCE:

Original fix https://github.com/TrinityCore/TrinityCore/commit/602f41c7ad541a888600f7692ccfaf061e92b740
## Tests Performed:

Tested on Windows 10 local server

## How to Test the Changes:

1. Kill a Firemane mob
2. Skin

## Known Issues and TODO List:

N/A

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Branel** [comment]
This port might not be as good as it seems! There might still be a inconsitency for droprates and some creatures might need a very close look, need a 2nd set of eyes on this.

File: data/sql/updates/pending_db_world/rev_1620897388675421400.sql [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+(60046,8170,0,20,0,1,1,1,2,"Rugged Leather"),
+(60047,4304,0,5,0,1,1,1,2,"Thick Leather"),
+(60047,8165,0,10,0,1,1,1,1,"Worn Dragonscale"),
+(60047,8170,0,80,0,1,1,1,2,"Rugged Leather"),
+(60047,8171,0,5,0,1,1,1,1,"Rugged Hide"),
+(60048,4304,0,40,0,1,1,1,2,"Thick Leather"),
+(60048,8165,0,10,0,1,1,1,1,"Worn Dragonscale"),
+(60048,8169,0,3,0,1,1,1,1,"Thick Hide"),
+(60048,8170,0,44,0,1,1,1,2,"Rugged Leather"),
+(60048,8171,0,3,0,1,1,1,1,"Rugged Hide");
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1620897388675421400.sql:141]: image

**Review by necropola — COMMENTED** [review]
- You are changing the skinning loot for **a lot** more creatures than just the **Firemanes**, but there is not a single comment or constant in the patch which would name these.
- Why are you deleting the existing skinning loot entries and introduce new ones (60028-60048)?
- Where is the source/evidence that these new values are correct?

**Comment by Branel** [comment]
This is a loot table taken directly from wowhead that was meant to resolve some common dragon skinning loot tables. The drop rates should be almost consistent with wowhead values depending on a case by case basis. That is why I pointed out it might not seem as good as it should. Most likely though most they are consistent and there are a few anomalies. When it comes to creature loot tables there are very few loot tables with actual comments, especially when many creatures share the same table.

**Comment by necropola** [comment]
> This is a loot table taken directly from wowhead that was meant to resolve some common dragon skinning loot tables. 
Post cataclysm wowhead data which in the case of **Dustwallow Marsh** isn't as bad as it would be for other zones but it's still not data from WotLK.

**Comment by Branel** [comment]
Its either data from classic or cata, and they are all at best guesswork. Pick your poison. 

The fix already exists and reduces any further need for these dragonkins to have to be reported on, hopefully.

**Comment by necropola** [comment]
> Its either data from classic or cata, and they are all at best guesswork. Pick your poison.

Did you try to actually find historic data, by using the history on Wowpedia or the wayback machine (web.archive.org) for wowhead? As an example: @Hacki95 went through the source code of old wowhead pages (from web.archive.org) to figure out what the drop rates for classic ores and stones actually were during WotLK.

**Comment by Branel** [comment]
Web archive is great, especially when you need to find old forum posts that detail spell mechanics and even old theorycrafting that might prove formulas for coefficients, but when it comes to droprates that might be the wrong way - once again depending on a case by case basis, some wowhead creatures might have a lot of data and its a perfect example of how things could work.

My question to you is, would it be right if i were to follow a archive that has maybe a couple 10k data collection and for some creatures <1k or a vanilla/cataclysm >100k data collection which might lead me in the right direction? I am happy to hear your opinion but I incline to go with the latter.

**Comment by necropola** [comment]
> * You are changing the skinning loot for **a lot** more creatures than just the **Firemanes**, but there is not a single comment or constant in the patch which would name these.
> * Why are you deleting the existing skinning loot entries and introduce new ones (60028-60048)?
> * Where is the source/evidence that these new values are correct?

I don't think that you have actually addressed any of my original concerns. Since I'm not in the position to approve PRs and i made my point, there isn't anything I could further contribute to this topic.

**Comment by Branel** [comment]
* The fix is imported from trinitycore where someone took the data from wowhead and used it to resolve numerous dragonkins skinning loot table, If I were to make this any more consistent then I rather implemenet the entire patch and this can possibly resolve  any further need to open up seperate reports on these other dragonkins as there might be issues with them that might arise with further testing, this saves the time of the developers and testers.
* The including and deleting anomaly will be resolved after I have resolved the build issues and have more time, see my first comment
* Wowhead is the source, the original fix documents this plentiful

Thank you for your time!

**Comment by necropola** [comment]
> after I have resolved the build issues 

I doubt that the `core-build / ubuntu-20.04-clang9` issue is caused by your patch. CI consistently fails with this error for some hours now, regardless of what the patch is.

**Comment by Branel** [comment]
I'll minimize this patch to halt any further discussion and open up a seperate PR after I have checked the other creatures and made sure that there is actually a need to change their loot table.

**Comment by offl** [comment]
> We're talking about a 5-10% margin for error here

5-10% is a lot, maybe then better to make it on your own? And use

> data from classic or cata

which even cannot be used as primary source

> https://github.com/azerothcore/azerothcore-wotlk/pull/5822/commits/64fe4718517b3424835257f122da20b6bb7f8f53

Nah, that's something that happens when you port something without having previous changes

**Comment by Branel** [comment]
I'll take your comment as a bad joke.

**Pull request closed without merging**